src/Controls/tests/DeviceTests/Elements/CollectionView/CollectionViewTests.iOS.cs

```diff
@@ -116,12 +117,14 @@ await InvokeOnMainThreadAsync(() =>
 				});
 
 				Assert.NotNull(cell);
+				collectionViewReference = new WeakReference(collectionView);
+				collectionView = null;
 			}
 
 			// HACK: test passes running individually, but fails when running entire suite.
 			// Skip the assertion on Catalyst for now.
 #if !MACCATALYST
-			await AssertionExtensions.WaitForGC(labels.ToArray());
+			await AssertionExtensions.WaitForGC(collectionViewReference);
```

Comment:
After these changes, when would the `labels` go away? Should we update this to check all the `VerticalCell` instances go away instead?

Reply:
If the `VerticalCell` was not created synthetically and held in local variable then they should be gone with the collection view.

Reply:
(The `ClearingItemsSourceClearsBindingContext` test was moved/renamed - it was originally added in https://github.com/dotnet/maui/pull/14619/files#diff-a1aed29ff55ec6fe5da9c27f12285c91406be414338738b0bcc3106ec4aa675f)

Reply:
I think it's already tested by this:
https://github.com/dotnet/maui/blob/639fcb0105a7cc966ecd8f7fe8277632fc6812e0/src/Controls/tests/DeviceTests/Elements/CollectionView/CollectionViewTests.cs#L106-L169